src/test/deploy/functions/checkIam.spec.ts

```diff
@@ -21,7 +21,7 @@ const BINDING = {
 const SPEC = {
   region: "us-west1",
   project: projectNumber,
-  runtime: "nodejs14",
+  runtime: "nodejs14" as const,
```

Comment:
TIL that `as const` satisfies enums without having to import them - good to know!

Reply:
yes, "as const" will make a string literal type rather than just a string type.